packages/yourloops/components/consent.tsx

```diff
@@ -0,0 +1,272 @@
+
+import brandingLogo from "branding/logo.png";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import FormControl from "@material-ui/core/FormControl";
+import FormHelperText from "@material-ui/core/FormHelperText";
+
+import DiabeloopUrl from "../lib/diabeloop-url";
+import { useAuth } from "../lib/auth";
+
+interface ConsentProps {
+  messageKey: string;
+  destinationPath: string;
+}
+
+const style = makeStyles((theme: Theme) => {
+  return {
+    mainContainer: { margin: "auto" },
+    Typography: {
+      alignItems: "center",
+    },
+    Card: {
+      display: "flex",
+      flexDirection: "column",
+      justifyContent: "center",
+      textAlign: "center",
+      // eslint-disable-next-line no-magic-numbers
+      padding: theme.spacing(4),
+    },
+    CardContent: {
+      marginLeft: theme.spacing(2),
+      marginRight: theme.spacing(2),
+    },
+    FormControl: {
+      margin: theme.spacing(3),
+    },
+    FormHelperText: {
+      textAlign: "center",
+    },
+    FormControlLabel: {
+      alignItems: "start",
+      textAlign: "start",
+      marginTop: theme.spacing(2),
+      marginBottom: theme.spacing(2),
+      marginLeft: theme.spacing(1),
+      marginRight: theme.spacing(1),
+    },
+    Buttons: {
+      display: "flex",
+      flexDirection: "row",
+      marginTop: theme.spacing(2),
+      marginRight: theme.spacing(5), // eslint-disable-line no-magic-numbers
+      marginLeft: theme.spacing(1), // eslint-disable-line no-magic-numbers
+    },
+    Button: {
+      marginLeft: "auto",
+    },
+  };
+});
+
+/**
+ * Patient Consent Page
+ */
+function Consent(props: ConsentProps): JSX.Element {
+  const { t, i18n } = useTranslation("yourloops");
+  const auth = useAuth();
+  const historyHook = useHistory();
+  const classes = style();
+  const [terms, setTerms] = React.useState(false);
+  const [privacyPolicy, setPrivacyPolicy] = React.useState(false);
+  const [error, setError] = React.useState(false);
+  const [helperText, setHelperText] = React.useState("");
+
+  const linkTermsText = t("Terms of Use");
+  const linkTerms = DiabeloopUrl.getTermsLink(i18n.language);
+  const privacyPolicyText = t("Privacy Policy");
+  const linkPrivacyPolicy = DiabeloopUrl.getPrivacyPolicyLink(i18n.language);
+
+  const resetFormState = (): void => {
+    setError(false);
+    setHelperText("");
+  };
+
+  const onChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>,
+    setState: React.Dispatch<React.SetStateAction<boolean>>
+  ): void => {
+    setState((event.target as HTMLInputElement).checked);
+    resetFormState();
+  };
+
+  const valideForm = (): boolean => {
+    if (terms && privacyPolicy) {
+      return true;
+    }
+
+    setError(true);
+    setHelperText("required");
```

Comment:
```suggestion
    setHelperText(t("required"));
```

Reply:
not needed is it is done here:
`                  <FormHelperText className={classes.FormHelperText}>
                    {t(helperText)}
                  </FormHelperText>`